### PR TITLE
Filtering results for review

### PR DESCRIPTION
**Pull request by jpotter-DFE**

### CODE REVIEW

File: DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageToolPage.cshtml [hunk 1]
```diff
@@ -7,10 +7,10 @@
         <li class="dfc-code-search-resultitem triage-search-result-item info-card" id="@pageItem.">
             <div class="info-card-content">
                 <h2 class="govuk-heading-m"><a class="govuk-link" href="@pageItem.PageLocation.FullUrl">@pageItem.DisplayText</a></h2>
-                @if (!string.IsNullOrWhiteSpace(pageItem.TriageToolSummary.Html))
+              @*   @if (!string.IsNullOrWhiteSpace(pageItem.TriageToolSummary.Html))
                 {
                     @Html.Raw(pageItem.TriageToolSummary.Html)
-                }
+                } *@
```
Comment (hnandra-DFE) [inline DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageToolPage.cshtml:13]: Can remove comment if code no longer necessary.

File: DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageFilterItem.cshtml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ﻿@model TriageToolFilterViewModel
 
 <div class="govuk-checkboxes__item">
-    <input class="govuk-checkboxes__input" id="@Model.Title" type="checkbox" value="@Model.Url" name="@nameof(TriageToolOptionViewModel.Filters)" checked="@Model.Selected">
+    @* <input class="govuk-checkboxes__input" id="@Model.Title" type="checkbox" value="@Model.Url" name="@nameof(TriageToolOptionViewModel.Filters)" checked="@Model.Selected">
     <label class="govuk-label govuk-checkboxes__label" for="@Model.Title">
         @Model.Title
-    </label>
+    </label> *@
```
Comment (hnandra-DFE) [inline DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageFilterItem.cshtml:7]: Can remove comment if code no longer necessary.

File: DFC.App.Triagetool/Properties/launchSettings.json [hunk 3]
```diff
@@ -17,11 +9,18 @@
     },
     "DFC.App.Triagetool": {
       "commandName": "Project",
-      "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+      "applicationUrl": "https://localhost:6001;http://localhost:6000"
```
Comment (bmatthews-DFE) [inline DFC.App.Triagetool/Properties/launchSettings.json:15]: @hnandra-DFE do we know the impact of this on other services?

File: DFC.App.Triagetool/Views/Pages/Body.cshtml [hunk 4]
```diff
@@ -1,74 +1,63 @@
...
-                    <div class="results-block">
-                        <div class="inner-block">
-                            @(await Html.PartialAsync("TriageToolPartialViews/TriageToolPage", Model))
-                        </div>
+
+                            }
+                        }
+                     </div>
+                     <div>
+                <button type="submit" class="govuk-button dfc-app-triage-btn ncs-button__primary govuk-button--start jscript-disabled-btn" id="applyFiltersJs" name="FilterAction" value="ApplyFilters">Apply Filters</button>
```
Comment (bmatthews-DFE) [inline DFC.App.Triagetool/Views/Pages/Body.cshtml:49]: Duplicated HTML buttons?

File: DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+            @if (facImage?.ImageHtml != null && !string.IsNullOrEmpty(facImage.ImageHtml.Html))
+            {
+                @Html.Raw(facImage.ImageHtml.Html);
+            }
+                
+            @foreach (var resultPage in Model.Pages.
+           Where(x => x.FilterAdviceGroup?.ContentItems != null && x.FilterAdviceGroup.ContentItems.Any(x => x.ContentItemId == filterAdviceGroup.ContentItemId)).OrderBy(x => x.TriageOrdinal))
+            {
+                Console.WriteLine(resultPage.ContentItemId);
```
Comment (bmatthews-DFE) [inline DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml:29]: @hnandra-DFE console log lines?

File: DFC.App.Triagetool/Properties/launchSettings.json (hunk 3 above)
Comment (hnandra-DFE) [reply DFC.App.Triagetool/Properties/launchSettings.json:15]: This has no impact on other services - only changes the port when running locally. Probably due to ports clashing when debugging multiple applications at the same time. I don't believe this code change is required though.

File: DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml (hunk 5 above)
Comment (hnandra-DFE) [reply DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml:29]: Console log lines should be removed.

File: DFC.App.Triagetool/Views/Pages/Body.cshtml [hunk 6]
```diff
@@ -1,74 +1,63 @@
...
-                <div class="filters" id="allSecondaryFilterAccordions">
-                    <div id="allSecondaryFilterAccordions">
-                        <div class="secondaryFilters" id="secondaryFilters1">
-                            <div id="secondaryFiltersSet1">
-                                <div class="govuk-form-group">
-                                    <fieldset class="govuk-fieldset">
-                                        <legend class="govuk-label govuk-!-font-weight-bold" id="secondaryFiltersHeading1">
-                                            Filter results
-                                        </legend>
+<form method="get" novalidate="novalidate" action="/triagetool?triage-level-one=@Model.SelectedLevelOne&triage-level-two=@Model.SelectedLevelTwo" id="TriageToolFilters">
```
[the remark this answers is not in the document]
Comment (siva-muruganandham) [reply DFC.App.Triagetool/Views/Pages/Body.cshtml:5]: done

File: DFC.App.Triagetool/Views/Pages/Body.cshtml (hunk 4 above)
Comment (siva-muruganandham) [reply DFC.App.Triagetool/Views/Pages/Body.cshtml:48]: They aren't duplicate. one for JS script disabled

File: DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml (hunk 5 above)
Comment (siva-muruganandham) [reply DFC.App.Triagetool/Views/Pages/TriageToolPartialViews/TriageResult.cshtml]: done

**Pull request merged**